src/gc/gcinterface.d

```diff
@@ -0,0 +1,196 @@
+/**
+ * Contains the internal GC interface.
+ *
+ * Copyright: Copyright Digital Mars 2005 - 2016.
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Jeremy DeHaan
+ */
+
+ /*          Copyright Digital Mars 2005 - 2016.
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ */
+module gc.gcinterface;
+import gc.stats;
+
+private
+{
+    static import core.memory;
```

Comment:
I agree, if `BlkInfo` is part of the interface (`qalloc` uses it), it should be defined here as well. But I'd avoid broadening the impact of this PR, let's defer this to another PR.